boar_bar/urls.py

```diff
@@ -33,6 +34,8 @@
     url(r'^user/', include('user.urls', namespace='user')),
     url(r'^admin/', admin.site.urls),
 
+    url(r'^api/similar_words/$', views_api.similar_words, name="similar_words"),

```

Comment:
I think it would be cleaner if you move this into api.urls

Reply:
ok